R/MIRA.R

```diff
@@ -56,93 +57,95 @@ if(getRversion() >= "2.15.1"){
-#' exBinDT=returnMIRABins(exampleBSDT,exampleRegionSet)
-returnMIRABins = function(BSDT,GRList, binNum=11, minReads = 500, 
-                          sampleNameInBSDT=TRUE){
+#' data("GM06990_1_ExampleSet", package = "MIRA") #exampleBSDT
+#' data("Gm12878Nrf1_Subset", package = "MIRA") #exampleRegionSet
+#' exBinDT = returnMIRABins(exampleBSDT, exampleRegionSet)
+returnMIRABins = function(BSDT, GRList, binNum = 11, minReads = 500, 
+                          sampleNameInBSDT = TRUE){
   
     #########returnMIRABins:Preprocessing and formatting###############
     #converting to list format if GRList is a data.table or GRanges object
-    if (class(GRList) %in% "GRanges"){
-      GRList=GRangesList(GRList)
+    if (class(GRList) %in% "GRanges") {
+      GRList = GRangesList(GRList)
       message("Converting to GRangesList...")
     }
-    if (class(GRList) %in% "data.table"){
-        GRList=list(GRList)
+    if (class(GRList) %in% "data.table") {
+        GRList = list(GRList)
         message("Converting to list...")
     }
 
     #checking that input is in list format
-    if (!class(GRList) %in% c("list","GRangesList")){
+    if (!class(GRList) %in% c("list", "GRangesList")) {
         stop("GRList should be a named list/GRangesList.")
     }
 
     #checking if region sets have names
-    if (is.null(names(GRList))){
+    if (is.null(names(GRList))) {
         warning("GRList should be a named list/GRangesList. 
                 Sequential names given according to order in object.")
-        names(GRList)<-paste0(rep("RegionSet",length(GRList)),1:length(GRList))
+        names(GRList) <- paste0(rep("RegionSet", length(GRList)), 
+                              1:length(GRList))
     }
 
     #checking that all objects in GRList are the same type 
     #and converting to data.tables
-    if (all(sapply(X = GRList,FUN = class) %in% "GRanges")){
+    if (all(sapply(X = GRList, FUN = class) %in% "GRanges")) {
         #GRanges to data.tables
-        GRDTList=lapply(X = GRList,FUN = grToDt,includeStrand=TRUE)
+        GRDTList = lapply(X = GRList, FUN = grToDt, includeStrand = TRUE)
         #below statement will be true if all objects in the list are of 
         #class data.table
         #necessary since data.tables also include data.frame as a class
     }else if (all(sapply(
-        X=lapply(X=GRList,FUN=function(x) class(x) %in% "data.table"),
-        FUN=any))){
+        X = lapply(X = GRList, FUN = function(x) class(x) %in% "data.table"), 
+        FUN = any))) {
         
-        GRDTList=GRList #this case is okay
+        GRDTList = GRList #this case is okay
     }else{
         stop("GRList should be a GRangesList or a list of data.tables")
     }
     
-    if (sampleNameInBSDT){
-        if(!"sampleName" %in% colnames(BSDT)){
-            stop("BSDT should have sampleName col if sampleNameInBSDT=TRUE")
+    if (sampleNameInBSDT) {
+        if (!"sampleName" %in% colnames(BSDT)) {
```

Comment:
Is the operator binding such that `if (!a %in% b)` always evaluates the same as `if (!(a %in% b))`? Regardless, I think that it's slightly clearer with parentheses, but just preference.